### ISSUE TITLE
Validation Guide Improvements

### ISSUE THREAD

**Issue by drjfloyd** [firemodels/fds#3806]
Craig commented to me that he felt the current guide was a bit lacking in terms of input descriptions for some of the experiments:

"Validation must be of a complete method, including the model itself and a fully specified means for developing inputs to the model. In the old days, FDS validation cases were fairly straightforward input sets and included the geometry, some material properties, and simple combustion parameters. Today, some of the validation cases have very complex input requirements. Without a complete description of how these inputs are to be determined, it is impossible to know what method is being validated. As such, the validation case is not of value to the user because he/she has no way of using FDS in the same way it was in the validation cases. While I realize the definition of the methods of defining inputs can be overwhelming at times, it is what is needed to move forward."

I think this is a fair comment. We should make sure for cases with more complex inputs that we either provide a rationalization for those inputs in the guide or make sure we point to some other publication where that rationalization sits.

**Comment by mcgratta** [comment]
I'll start this ball rolling by going back over the validation cases I've done. Once I determine a format for providing inputs, I'll ask others to do same.

**Comment by mcgratta** [comment]
I have submitted this is a summer project for a NIST SURF (Summer Undergraduate Research Fellowship).

**Comment by mcgratta** [comment]
I did not get my summer student, but I have started working this issue. For validation cases that I am currently working on, I add "Modeling Notes" to the end of the experimental description. Details of how we model the experiments should be added in this section. In the rest of the guide, add specific details about the results presented, like where the measurement devices were located, but put all other details in the Experiment chapter under "Modeling Notes".
